backend/app/assets/stylesheets/spree/backend/components/_messages.scss

```diff
@@ -55,3 +55,29 @@
 
   &.error a { color: very-light($color-error, 10) }
 }
+
+.deprecation-warning {
```

Comment:
This is not a good way to add css. These styles are only ever going to be used by the one box on the one page (and it doesn't even fit on that page particularly well). Deciding that `i` have a font size of 30px also makes no sense.